### DISCUSSION TITLE
Let's Encrypt certificate: missing "renew" button

### DISCUSSION THREAD

**Discussion by panagushin** [ravendb/ravendb#14979] [Q&A] [answered]
Hi, I've used Setup Wizard to get my certificate and free domain, and it's expire soon, but I don't see the renew button on certificates page. 

<img width="741" alt="image" src="https://user-images.githubusercontent.com/5909693/191169032-3b92226c-bd9e-4d19-a6a1-7d26cd993e2d.png">

Are there any workarounds to get this button or now I have only option to provide certificate manually?

**Comment by ayende** [comment]
What does your `settings.json` show?
Where are you running this?

Generally, if you are using Let's Encrypt, RavenDB will handle renewal automatically. A month ahead of time. So this is not supposed to happen.
Can you check the logs as well?

**Comment by panagushin** [comment]
Running: Ubuntu 20.04 + Docker

Settings.json

`
{
  "Security.UnsecuredAccessAllowed": "PrivateNetwork",
  "License.Eula.Accepted": true,
  "Setup.Mode": "Secured",
  "Security.Certificate.Path": "/opt/RavenDB/Server/cluster.server.certificate.{name}.ravendb.community.pfx",
  "ServerUrl": "https://423686a74436:8080",
  "ServerUrl.Tcp": "tcp://423686a74436:38888",
  "PublicServerUrl": "https://{name}.ravendb.community:8080",
  "PublicServerUrl.Tcp": "tcp://{name}.ravendb.community:38888"
}
`

`Can you check the logs as well?`
on which type of events? there are tons of logs in studio.

**Reply by ayende** [reply]
`"Setup.Mode": "Secured"`

That is the problem.
You are using a Let's Encrypt certificate, but it looks like you _imported_ it, rather than let RavenDB generate it.

We expect the value there to be `Setup.Mode: LetsEncrypt` when we manage that.

**Reply by panagushin** [reply]
I 100% sure I didn't import certificate, but probably did other manipulations so `Setup.Mode` changed to Secured :) 
Can I try to change `Setup.Mode` to `LetsEncrypt` ?

**Reply by ayende** [reply]
You are missing some required fields (such as email,etc).

I would recommend re-running the setup wizard, instead.

**Reply by panagushin** [reply]
great, how I can re-run the setup wizard?

**Comment by haludi** [comment, marked as answer]
You need to set Setup.Mode to Initial.
You can use the following as the content of settings.json

{
    "Setup.Mode": "Initial"
}

**Reply by panagushin** [reply]
Thanks, 
but I just added this fields to settings:
`"Security.Certificate.LetsEncrypt.Email": "{email which I used to in setup wizard}"`
`"ExternalIp": "{public IP of host with docker}"`
and updated `Setup.Mode` to `LetsEncrypt` 

and now I can renew cert :)
<img width="747" alt="image" src="https://user-images.githubusercontent.com/5909693/191250949-eb4710d8-0f59-4d1a-9e1e-b50046030f22.png">
